### PR TITLE
add caps and limits information

### PR DESCRIPTION
**Pull request by AlexBaranowski**
@tompods it should be short one.

Based mostly on the https://access.redhat.com/articles/rhel-limits

### CODE REVIEW

File: docs/knowledge-base/eurolinux-capabilities-and-limits.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+| x86_64  | 448 (4096)  |  768 (5120) | 768 (8192)  | TBA |
+| ARM64  | X  | X  | 256  | TBA  |
+<!-- | POWER  | X | X | TBA  | TBA  | -->
+
+
+## Minimum memory 
+
+These requirements are only for systems supported as VMs or Physical hosts. In
+most cases, it's possible to run a system without complications on less memory.
+The minimum requirements are in place because it might not be possible to
```
Comment (tompods) [inline docs/knowledge-base/eurolinux-capabilities-and-limits.md:42]: "are in place" -> "have been provided below"

**Review by tompods — CHANGES_REQUESTED** [review]

File: docs/knowledge-base/eurolinux-capabilities-and-limits.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+| Maximum File Size  | 8EiB | 8EiB | 8EiB | 8EiB |
+| Maximum Filesystem Size | 16EiB | 16EiB | 1PiB | 1PiB |
+| Maximum Subdirectories or files in directory | unlimited | unlimited | unlimited | unlimited |
+| Maximum symlink depth | 8 | 8 | 8 | 8 | 
+
+
+### GFS2
+
+Global Filesystem 2 is part of EuroLinux Resilient Storage and High
+Availability add-ons that are freely available with subscription or free
```
[the remark this answers is not in the document]
Comment (AlexBaranowski) [reply docs/knowledge-base/eurolinux-capabilities-and-limits.md:109]: RHEL and OL require additional subscriptions for HA and RS. When clients buy our products, we support HA and RS without additional cost. That's why I wrote freely available with a subscription. Also, it's part of the "true product" core company value.

**Comment by AlexBaranowski** [comment]
@tompods please review. You are the 🥇

**Comment by tompods** [comment]
Almost OK, just use the exact expression: "for the systems **for** which support is" (the "for" word in bold is missing)

**Comment by AlexBaranowski** [comment]
@tompods you can merge

**Review by tompods — APPROVED** [review]

**Pull request merged**